src/main/java/com/google/api/codegen/discogapic/transformer/java/JavaDiscoGapicSchemaToViewTransformer.java

```diff
@@ -168,7 +167,7 @@ private StaticLangApiMessageView generateSchemaClasses(
     // Getters and setters use unescaped name for better readability on public methods.
     schemaView.fieldGetFunction(context.getDiscoGapicNamer().getResourceGetterName(schemaId));
     schemaView.fieldSetFunction(context.getDiscoGapicNamer().getResourceSetterName(schemaId));
-    String schemaTypeName = schemaTypeTable.getAndSaveNicknameForElementType(schema);
+    String schemaTypeName = schemaTypeTable.getAndSaveNicknameFor(schema, true);
```

Comment:
Removing the boolean param for now, since all current invocations only pass in "true".